test/unit/metrics-recorder/http.test.js

```diff
@@ -383,6 +393,18 @@ describe("recordWeb", function() {
           {name: 'WebTransactionTotalTime/NormalizedUri/*'},
           [1, 0.001, 0.001, 0.001, 0.001, 0.000001]
         ],
+        [{name: 'DurationByCaller/Unknown/Unknown/Unknown/Unknown/all'},
+          [1, 0.001, 0.001, 0.001, 0.001, 0.000001]
+        ],
+        [{name: 'ErrorsByCaller/Unknown/Unknown/Unknown/Unknown/all'},
+          [1, 0.001, 0.001, 0.001, 0.001, 0.000001]
+        ],
+        [{name: 'DurationByCaller/Unknown/Unknown/Unknown/Unknown/allWeb'},
+          [1, 0.001, 0.001, 0.001, 0.001, 0.000001]
+        ],
+        [{name: 'ErrorsByCaller/Unknown/Unknown/Unknown/Unknown/allWeb'},
+          [1, 0.001, 0.001, 0.001, 0.001, 0.000001]
+        ],
```

Comment:
Minor: Test has nothing specific to do with DT so feels a bit superfluous to add these here (not the intent of the test).